composer.json

```diff
@@ -9,6 +9,7 @@
     "prefer-stable": true,
     "require": {
         "php": ">=5.6",
+        "symfony/config": "^4.2",
```

Comment:
According to the CI:
```
symfony/config v4.4.5 requires php ^7.1.3 -> your PHP version (7.0.33) does not satisfy that requirement.
```
It means that we need to update to `"php": ">=7.1", modify travis file accordingly, and create a major release

Reply:
Done